test/functional/rpc_signer.py

```diff
@@ -78,7 +78,7 @@ def run_test(self):
         self.clear_mock_result(self.nodes[1])
 
         result = self.nodes[1].enumeratesigners()
-        assert_equal(len(result['signers']), 2)
+        assert_equal(len(result['signers']), 1)
         assert_equal(result['signers'][0]["fingerprint"], "00000001")
         assert_equal(result['signers'][0]["name"], "trezor_t")
```

Comment:
This whole section can now be one line.
```diff
-        result = self.nodes[1].enumeratesigners()
-        assert_equal(len(result['signers']), 1)
-        assert_equal(result['signers'][0]["fingerprint"], "00000001")
-        assert_equal(result['signers'][0]["name"], "trezor_t")
+        assert_equal(self.nodes[1].enumeratesigners(), {'signers': [{'fingerprint': '00000001', 'name': 'trezor_t'}]})
```

Reply:
The reason I made this more verbose initially is because we might add extra fields. Maybe use a Python way to check "at least these fields are present and have the right value"?

Reply:
I took both points into account, please check my update